### PR TITLE
Fix ROS-Webots matching.

### PR DESCRIPTION
**Pull request by DavidMansolino**
**Description**
Follow-up of discussion https://github.com/cyberbotics/webots/pull/2191#discussion_r482723706

### CODE REVIEW

**Comment by DavidMansolino** [comment]
Note sure where it is the best location to put the documentation about usign the development version of Webots + development version of webots_ros:
  - wiki?
  - in a README file?
  - in the doc?

**Comment by omichel** [comment]
I would add it here: https://github.com/cyberbotics/webots/wiki and also http://wiki.ros.org/webots_ros

**Comment by DavidMansolino** [comment]
https://github.com/cyberbotics/webots/wiki/Testing-the-ROS-controller

**Comment by omichel** [comment]
We would also need a link from the above page.

**Comment by DavidMansolino** [comment]
> We would also need a link from the above page.

Just to summarize what we discussed on Discord, the link is already here: https://github.com/cyberbotics/webots/pull/2197#issuecomment-686350038

File: docs/guide/tutorial-8-using-ros.md [hunk 1]
```diff
@@ -117,5 +117,5 @@ You can find in the [Reference Manual](../reference/nodes-and-api-functions.md)
 ### "webots\_ros" Package
 
 If you are running the latest version of Webots, the easiest way to setup the `webots_ros` package is to install the `ros-$ROS_DISTRO-webots-ros` package directly from the package manager as described above.
-But if you are running an older Webots version, some functionality might not be fully supported by the latest `ros-$ROS_DISTRO-webots-ros` package.
-In this case, it is possible to install an older `webots_ros` package version from sources following the instructions on the [ROS wiki page](http://wiki.ros.org/webots_ros#From_Sources) and selecting the desired branch in the [`webots_ros` GitHub repository](https://github.com/cyberbotics/webots_ros).
+But if you are running an older Webots version, some functionality might not be fully supported or broken by the latest `ros-$ROS_DISTRO-webots-ros` package.
```
Comment (stefaniapedrazzi) [inline docs/guide/tutorial-8-using-ros.md:120]: Not sure, but I would say that the `might not be` also applies to `broken` in this sentence.
here is a suggestion that seems a little bit more clear to me (also found an old typo)
```suggestion
But if you are running an older Webots version, some functionalities might not be fully supported or might be broken by the latest `ros-$ROS_DISTRO-webots-ros` package.
```

File: docs/guide/tutorial-8-using-ros.md [hunk 2]
```diff
@@ -117,5 +117,5 @@ You can find in the [Reference Manual](../reference/nodes-and-api-functions.md)
...
+In this case, it is possible to install an older `webots_ros` package version from sources following the instructions on the [ROS wiki page](http://wiki.ros.org/webots_ros#From_Sources) and selecting the tag/release matching the Webots version in the [`webots_ros` GitHub repository](https://github.com/cyberbotics/webots/releases).
```
Comment (stefaniapedrazzi) [inline docs/guide/tutorial-8-using-ros.md:121]: Don't we want to link to the `webots_ros` repo?
```suggestion
In this case, it is possible to install an older `webots_ros` package version from sources following the instructions on the [ROS wiki page](http://wiki.ros.org/webots_ros#From_Sources) and selecting the tag/release matching the Webots version in the [`webots_ros` GitHub repository](https://github.com/cyberbotics/webots_ros/releases).
```

**Review by stefaniapedrazzi — CHANGES_REQUESTED** [review]
Thank you!

It's good to now be able to easily find the `webots_ros` package matching the Webots version.

File: docs/guide/tutorial-8-using-ros.md (hunk 1 above)
Comment (DavidMansolino) [reply docs/guide/tutorial-8-using-ros.md:120]: You're right, this looks better.

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you

**Pull request merged**